algorithms/kernel/dtrees/gbt/classification/gbt_classification_train_dense_default_impl.i

```diff
@@ -61,54 +61,62 @@ public:
         TVector<algorithmFPType, cpu, ScalableAllocator<cpu> > aExp(n);
         auto exp                           = aExp.get();
         const algorithmFPType expThreshold = daal::internal::Math<algorithmFPType, cpu>::vExpThreshold();
-        if (sampleInd)
-        {
-            PRAGMA_IVDEP
-            PRAGMA_VECTOR_ALWAYS
-            for (size_t i = 0; i < n; ++i)
+        const size_t nThreads              = daal::threader_get_threads_number();
+        const size_t nBlocks               = getNBlocksForOpt<cpu>(nThreads, n);
+        const size_t nPerBlock             = n / nBlocks;
+        const size_t nSurplus              = n % nBlocks;
+        const bool inParallel              = nBlocks > 1 ? true : false;
```

Comment:
No reason to have '? true : false' at all.

Reply:
Fixed